faq-bundle/src/Resources/contao/dca/tl_faq.php

```diff
@@ -520,6 +569,41 @@ public function generateAlias($varValue, DataContainer $dc)
 		return $varValue;
 	}
 
+	/**
+	 * Return the SERP URL
+	 *
+	 * @param FaqModel $objFaq
+	 *
+	 * @return string
+	 */
+	public function getSerpUrl(FaqModel $objFaq)
+	{
+		/** @var FaqCategoryModel $objCategory */
+		$objCategory = $objFaq->getRelated('pid');
+
+		if ($objCategory === null)
+		{
+			throw new Exception('FAQ category does not exist.');
+		}
+
+		$jumpTo = (int) $objCategory->jumpTo;
+
+		// A jumpTo page is not mandatory for FAQ categories (see #6226) but required for the FAQ list module
+		if ($jumpTo < 1)
+		{
+			throw new Exception('FAQ categories without redirect page cannot be used in an FAQ list');
```

Comment:
should this message be translated (dunno if thats the same for news etc)?

Reply:
This message is only shown if any third party extension uses the `getSerpUrl` manually. `tl_faq::checkSerpPreview` will remove the `serpPreview` field for faq categories without a redirect page.